xml/System.Text.Json/JsonElement+ArrayEnumerator.xml

```diff
@@ -172,8 +172,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.Generic.IEnumerator" /> object for <see cref="T:System.Text.Json.JsonElement" /> objects that can be used to iterate through the collection.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.Json.JsonElement.ArrayEnumerator> instance is cast to an [IEnumerable\<JsonElement>](<xref:System.Collections.Generic.IEnumerable%601) interface. 

         ]]></format>
         </remarks>
```

Reply:
Added the `## Remarks` header to your suggestion.